openssl/src/symm.rs

```diff
@@ -23,7 +23,7 @@
 //!     &ciphertext[..]);
 //! ```
 //!
-//! Encrypting an assymetric key with a symmetric cipher
+//! Encrypting an asymetric key with a symmetric cipher
```

Comment:
`asymmetric` :)

Reply:
Haha! This is what I get for trying to open a PR on my phone!